From c411ac250c57902b98cdd1302a41b04b531dccbf Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Tue, 24 Dec 2024 18:39:03 +0100
Subject: [PATCH 2/5] Update maintainer documentation for trusted.gpg.d files
 removal

These are now ASCII Armored files, and as such, use an .asc file
extension.
---
 README.maintainer | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/README.maintainer b/README.maintainer
index e10d989..1c85067 100644
--- a/README.maintainer
+++ b/README.maintainer
@@ -50,7 +50,7 @@ and checking the contents of each keyring
 
 Add an entry to debian/debian-archive-keyring.maintscript:
 
-rm_conffile /etc/apt/trusted.gpg.d/debian-archive-${foo}.gpg ${version}~~
+rm_conffile /etc/apt/trusted.gpg.d/debian-archive-${foo}.asc ${version}~~
 
 Pre-build
 ---------
-- 
2.47.1

